cic-ipv-stub/template.yaml

```diff
@@ -494,15 +493,14 @@ Resources:
       Tags:
         APP: !Sub cic-${AWS::StackName}
 
-
 Outputs:
-  APIGWExecuteUrl:
+  IPVStubExecuteUrl:
     Description: "API Gateway endpoint URL for the environment stage"
     Value: !Sub "https://${IPVStubApiGw}.execute-api.${AWS::Region}.amazonaws.com/${Environment}/"
     Export:
-      Name: !Sub "${AWS::StackName}-APIGWExecuteUrl"
-  APIGWID:
+      Name: !Sub "${AWS::StackName}-IPVStubExecuteUrl"
+  IPVStubApiGw:
     Description: "The ID of the API Gateway"
     Value: !Ref IPVStubApiGw
     Export:
-      Name: !Sub "${AWS::StackName}-APIGWID"
\ No newline at end of file
+      Name: "cic-cri-IPVStubApiGw"
```

Comment:
This will need to include the stack name right? If we deploy more than one of this stack into an environment, we'll end up with naming collisions.

Reply:
Having spoken to @devdutoitGDS guidance on this is if we need multiple deployed stubs created multiple records in the dns template to have routes for the different stubs but the idea is we'll only have one 'working' stub on the account and the stubs being deployed for testing purposes can just be tested via api testing without a domain?